jenkins-scripts/docker/lib/_ros_setup_buildsh.bash

```diff
@@ -23,8 +23,8 @@ if ${USE_CATKIN_MAKE}; then
 else
   export CMD_CATKIN_CONFIG=""
   export CMD_CATKIN_LIST="colcon list -t"
-  export CMD_CATKIN_BUILD="colcon build --parallel-workers ${MAKE_JOBS} --symlink-install --event-handler console_direct+ ${CATKIN_EXTRA_ARGS}"
-  export CMD_CATKIN_TEST="colcon test --parallel-workers 1 --event-handler console_direct+ || true"
+  export CMD_CATKIN_BUILD="colcon build --parallel-workers ${MAKE_JOBS} --symlink-install --event-handler console_cohesion+ ${CATKIN_EXTRA_ARGS}"
+  export CMD_CATKIN_TEST="colcon test --executor sequential --event-handler console_direct+ || true"
```

Comment:
Actually why are you using `| true` here? I don't think that is desired. The return code only indicates if the tests were involved correctly - not indicating anything about test results.

Reply:
thanks fixed in a23773f